api_tests/conftest.py

```diff
@@ -14,3 +16,28 @@ def app():
 @pytest.fixture(autouse=True, scope='session')
 def app_init():
     init_app(routes=False, set_backends=False)
+
+
+@pytest.fixture(scope='function')
+def es_client():
```

Comment:
Minor, but for disambiguation this seems like the sort of thing that should maybe have a `6`